pom.xml

```diff
@@ -282,6 +282,7 @@
                         <configuration>
                             <sources>
                                 <source>src/main/scala</source>
+								<source>${project.build.directory}/generated-sources</source>
```

Comment:
indention looks incorrect. We are using spaces only. Maybe your IDE renders tabs differently...